lib/commands/publish.js

```diff
@@ -156,11 +158,18 @@ class Publish extends BaseCommand {
       }
     }
 
-    const latestVersion = await this.#highestPublishedVersion(resolved, registry)
-    const latestSemverIsGreater = !!latestVersion && semver.gte(latestVersion, manifest.version)
+    if (!force) {
+      const { highestVersion, versions } = await this.#registryVersions(resolved, registry)
+      const latestSemverIsGreater = !!highestVersion && semver.gte(highestVersion, manifest.version)
 
-    if (latestSemverIsGreater && isDefaultTag) {
-      throw new Error(`Cannot implicitly apply the "latest" tag because published version ${latestVersion} is higher than the new version ${manifest.version}. You must specify a tag using --tag.`)
+      if (versions.includes(manifest.version)) {
+        throw new Error(`You cannot publish over the previously published versions: ${manifest.version}.`)
```

Comment:
```suggestion
        throw new Error(`You cannot publish over the previously published version: ${manifest.version}.`)
```

Reply:
this is parity with registry error and was intentionally left pluralized

Reply:
well ok then!